packages/workflow-app-textedit/src/index.js

```diff
@@ -0,0 +1,30 @@
+/* global Application */
+const safari = {
+  type: 'app',
+  name: 'Textedit',
+  params: ['file'],
+  open: async ({ file, position }, { platform, run }) => {
+    if (platform !== 'osx') {
+      throw new Error('Platform is not supported');
+    }
+
+    return run(
```

Comment:
I'm not on a Mac today, but I remember I had some issues with making TextEdit behave consistently on OSX. 

Have you ensured that these are all working as intended?
1. start when `TextEdit` is not opened at all
2. start when `TextEdit` is open without any windows
3. start when `TextEdit` is open with a minimized window
4. start when `TextEdit` is open with visible window

All of this should create a new window and bring it to the front.

Reply:
Not 3., can test that after I have fixed the changes. It seems to work fine, but have the same inconsistencies that other things have on macos with several windows.